examples/simulation/Tutorial2/macros/run_tutorial2.C

```diff
@@ -12,7 +12,8 @@
 #include <TSystem.h>
 #include <memory>
 
-void run_tutorial2(Int_t nEvents = 10, TString mcEngine = "TGeant4", Bool_t isMT = true)
+void run_tutorial2(Int_t nEvents = 10, TString mcEngine = "TGeant4",
+                   Bool_t isMT = true, UInt_t initial_seed=98989)
```

Comment:
Please update the copyright header on this file as well.

Reply:
Done. I added also missing header files. Since I was working on the issue anyway I also added the missing header files in all other macros of Tutorial2.